.gitlab-ci.yml

```diff
@@ -656,15 +656,12 @@ undefinedsanitizer:
       -DCTEST_MEMORYCHECK_TYPE=UndefinedBehaviorSanitizer
 
 cudamemcheck:
-  stage: QoS_tools
+  stage: build
```

Comment:
should this stay?

Reply:
I would like to ensure that it is working before I merge this.
But I wonder whether it exceed the time limit of slurm.

Reply:
I will reverse it before merging when the job is finished once.

Reply:
It is passed. I have also restructured the commit history with reversing the temporary check.